data/service/api/v1/alerts.go

```diff
@@ -36,20 +38,19 @@ func DeleteAlert(dCtx service.Context) {
 		return
 	}
 
-	if err := checkUserIDConsistency(authDetails, r.PathParam("userID")); err != nil {
+	path := getUserIDsFromPath(r)
```

Comment:
[nit] Consider renaming `path` to `pathUserIDs`. I was confused by "getUserIDsFromPath" returning the "path".